app.rb

```diff
@@ -49,6 +49,9 @@ def config
 
       ial = prepare_step_up_flow(session: session, ial: ial, aal: aal)
 
+      is_eipp_allowed = is_eipp_allowed()
+      ial_select_options = get_ial_select_options(is_eipp_allowed: is_eipp_allowed)
```

Comment:
1. The empty parens for the method call jumps out at me, would definitely avoid that
2. See other comments, but I think we don't need to pass in `is_eipp_allowed`
3. I bet we could remove this entirely and just call the method inline in the hash below
```suggestion
```

Reply:
I removed the method entirely and called the method inline in the hash with commit db8c6741ec7d24e4524248238bab00ae45f1fbc4.
That reads nicer.